### PR TITLE
Rename `node --master` -> `node --leader`

### PR DESCRIPTION
**Pull request by branden**
See https://mesosphere.atlassian.net/browse/DCOS-5441.

### CODE REVIEW

**Comment by tamarrow** [comment]
@branden can we add a deprecation notice for `--master`?

File: cli/dcoscli/node/main.py [hunk 1]
```diff
@@ -24,9 +25,16 @@ def main():
 @decorate_docopt_usage
 def _main():
     util.configure_process_from_environ()
+    argv = sys.argv[1:]
```
Comment (branden) [inline cli/dcoscli/node/main.py]: I'd much rather let `docopt.docopt()` decide what `argv` is, but I'm not sure how else to add this deprecation notice without adding `--master` back to the usage string.

Comment (tamarrow) [inline cli/dcoscli/node/main.py]: yes, sadly you have to also add `--master` back into the usage string.

**Comment by tamarrow** [comment]
@branden the tests failed from a syntax error: dcoscli/node/main.py:3:1: F401 'sys' imported but unused

**Pull request merged**